src/main/java/zabi/minecraft/nbttooltip/config/ConfigInstance.java

```diff
@@ -33,15 +33,15 @@ public ConfigInstance() {
 		copyingEngine = CopyingEngine.JSON;
 	}
 	
-	public static enum TooltipEngine {
+	public enum TooltipEngine {
```

Comment:
Is there any reason for this change? Because it currently breaks other stuff I'm working with

Reply:
![image](https://user-images.githubusercontent.com/26803198/121932884-acc59700-cd45-11eb-9270-6fa7dde626b4.png)
Should I still change it back?

Reply:
This one yes, I need this reverted, I use the "static" keyword as a marker in another project of mine and in need it to be explicit in this case